### ISSUE TITLE
Events present in the documentation not working

### ISSUE THREAD

**Issue by abeersaqib** [iVis-at-Bilkent/cytoscape.js-expand-collapse#30]
```js 
cy.nodes().on("expandcollapse.afterexpand", function(event) { var node = this; ... })
```
won't worked while
```js 
cy.nodes().on("afterExpand", function(event) { var node = this; ... })
```
worked fine.

in your code you are triggering  node.trigger("afterExpand"); 

Please update your documentation :)

Thanks !

**Comment by metincansiper** [comment]
@abeersaqib I tested it seems that
```js 
cy.nodes().on("expandcollapse.afterexpand", function(event) { var node = this; ... })
```
is working. 

I think that I forgot to update browserified file before releasing v3.0.0. If you are using v3.0.0 you may have this problem. However, it must be fixed in v3.0.1. 

If v3.0.1 is working well for you we can close the issue.

**Comment by abeersaqib** [comment]
@metincansiper we have bit modified your library since there were memory reference issues when library is used in Desktop mode instead of browser mode.
I don't know whether those changes were added in version 3.0.1 or not.
